### PR TITLE
Feature interactive list

### PR DESCRIPTION
**Pull request by webdevmaxi**
### Add Drag/Drop and Storage functionality.
- Items get sorted and re-positioned when dropped on others.
- List of items and sort order is persisted in local storage.
## Notes
- In Firefox drag & drop doesn't work properly. Tested to work on Chromium/Ubuntu.
- Items don't get shifted until dropped. Drag-over event is not doing the trick.
- I get an eslint error saying: "Expected 'this' to be used by class getter 'taskListComponent'" that I haven't been able to resolve.

### CODE REVIEW

File: src/completed.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      if (element.index === idx) {
+        if (element.completed === true) {
+          element.completed = false;
+        } else if (element.completed === false) {
+          element.completed = true;
+        }
+      }
+    });
+  }
+}
```
Comment (Dipeshtwis) [inline src/completed.js:14]: - [ ]  The completed task array of items is not stored in local storage, currently after refreshing all the completed task would come back to previous state. Make sure to reload the page to test the array is stored.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @webdevmaxi, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: It has event listeners on the list items (`dragstart`, `dragover`, `drop`) and to checkbox (`change`).


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by webdevmaxi** [comment]
I've added a function call to in the updateComplete function to make the checked status of the task item persistent on the local storage.

**Review by MahmoudBakr23 — COMMENTED** [review]
Hi team 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**